SQL_DDL_HW

crisforp opened the pull request:
@mozuela 
Task: create a physical DB

crisforp committed:
Add files via upload

Task: create a physical DB

mozuela commented:
Hi Cristian, very good work, all constrains were implemented including not null, check, foreign key with detailed comments.

My only recommendations will be trying to implement at least one GENERATED ALWAYS AS to practice and to include WHERE NOT EXISTS, IF NOT EXISTS in those inserts without on conflict constraint.

crisforp commented:
> Hi Cristian, very good work, all constrains were implemented including not null, check, foreign key with detailed comments.
> 
> My only recommendations will be trying to implement at least one GENERATED ALWAYS AS to practice and to include WHERE NOT EXISTS, IF NOT EXISTS in those inserts without on conflict constraint.

Thank you for your nice words and for your strong recommendations. I will make the changes and send them to you as soon as I can.

crisforp committed:
Add files via upload

My only recommendations will be trying to implement at least one GENERATED ALWAYS AS to practice and to include WHERE NOT EXISTS, IF NOT EXISTS in those inserts without on conflict constraint.
